docs/planning/frontend/src/pages/ImportPage.jsx

```
@@ -0,0 +1,257 @@
+      {/* Seed Reviewers */}
+      <div className="card mb-6">
+        <h2 className="text-xl font-semibold mb-4">Family Members</h2>
+        <p className="text-gray-600 mb-4">
+          Create the default family member accounts: Byron, Veronica, Bayden, and Ariannah.
+        </p>
+
+        <button
+          onClick={() => seedReviewersMutation.mutate()}
+          disabled={seedReviewersMutation.isPending}
+          className="btn btn-primary"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)